docs/docs/concepts/policies.mdx

```diff
@@ -378,41 +343,44 @@ The _Intentless Policy_ is part of Rasa's new Conversational AI with
-
-This only requires appropriate responses to be defined in the domain file.
-
-To eliminate hallucinations, the policy only chooses which response from
-your domain file to send. It does not generate new text.
-
-In addition, you can control the LLM by:
-
-- providing example conversations (end-to-end stories) which will be used in the prompt.
-- setting the confidence threshold to determine when the intentless policy should kick in.
-
-[This repository](https://github.com/RasaHQ/starter-pack-intentless-policy) contains a starter pack with a bot that uses the
-`IntentlessPolicy`. It's a good starting point for trying out the policy and for
-extending it.
-
-## Demo
-
-[Webinar demo](https://hubs.ly/Q01CLhyG0) showing that this policy can already
-handle some advanced linguistic phenomena out of the box.
-
-The examples in the webinar recording are also part of the end-to-end tests
-defined in the [example repository](https://github.com/RasaHQ/starter-pack-intentless-policy) in (`tests/e2e_test_stories.yml`).
-
-## Adding the Intentless Policy to your bot
+The Intentless Policy is used to send `responses` in a conversation which
+are not part of a flow. 
+This can be helpful for handling chitchat, contextual questions,
+and high-stakes topics.
+
+### Chitchat
+In an enterprise setting it may not be appropriate to use
+a purely generative model to handle chitchat.
+Teams want to ensure that the assistant is always on-brand and on-message.
+Using the Intentless Policy, you can define vetted, human-authored `responses`
+that your assistant can send.
+Because the Intentless Policy leverages LLMs and considers the whole context of the conversation
+when selecting an appropriate `response`, it is much more powerful
+than simply predicting an intent and triggering a fixed response to it. 
+
+### High-stakes Topics
+In addition to chitchat, another common use case for the Intentless Policy is
+providing answers on high-stakes topics.
+For example, if users have questions about policies, legal terms, or guarantees,
+like: _"My situation is X, I need Y. Is that covered by my policy?"_
+In these cases the DocSearchPolicy's RAG approach is risky. 
+Even with the relevant content present in the prompt, a RAG approach allows an 
+LLM to make interpretations of documents. 
+The answer users get will vary depending on the exact phrasing of their question,
+and may change when the underlying model changes. 
+
+For high-stakes topics, it is safest to send a self-contained, vetted answer
+rather than relying on a generative model.
+The Intentless Policy provides that capability in a CALM assistant.
+
+### Interjections
+When a flow reaches a `collect` step and your assistant asks the user for information,
+your user might ask a clarifying question, refuse to answer, or otherwise interject in 
+the continuation of the flow. 
+In these cases, the Intentless Policy can contextually select appropriate `responses`,
+while afterwards allowing the flow to continue.
```

Comment:
clarifying questions might be more the domain of doc search - hard to tell

Reply:
An example here would help the reader to picture what sort of a clarifying question we are talking about.